bin/setenv.sh

```diff
@@ -124,23 +126,19 @@ else
   xmlstarlet ed -L -s "/Server/Service/Connector" -t attr -n "maxThreads" -v "${EXO_HTTP_THREAD_MAX}" \
     -s "/Server/Service/Connector" -t attr -n "minSpareThreads" -v "${EXO_HTTP_THREAD_MIN}" \
     ${CHAT_APP_DIR}/conf/server.xml || {
-    echo "ERROR during xmlstarlet processing (adding Connector proxyName)"
+    echo "ERROR during xmlstarlet processing (adding Connector thread pool configuration)"
+    exit 1
+  }
+
+  # Tomcat HTTP port configuration
+  xmlstarlet ed -L -u '/Server/Service/Connector/@port' -v "${CHAT_HTTP_PORT}" \
+    -u "/Server/@port" -v "${CHAT_SERVER_STOP_PORT}" ${CHAT_APP_DIR}/conf/server.xml || {
+    echo "ERROR during xmlstarlet processing (changing Connector port)"
     exit 1
   }
 
   # JMX configuration
   if [ "${EXO_JMX_ENABLED}" = "true" ]; then
-    # insert the listener before the "Global JNDI resources" line
```

Comment:
The JMX is used by us. Please keep the configuration

Reply:
I know that you use it, in fact this configuration is deprecated in Tomcat and will be removed as [mentioned here](https://tomcat.apache.org/tomcat-8.5-doc/config/listeners.html#JMX_Remote_Lifecycle_Listener_-_org.apache.catalina.mbeans.JmxRemoteLifecycleListener)

In addition, in PRD 6.1 we will upgrade to Tomcat 9, so I will revert this until we 'll do the upgrade in product and let you re-work it.

(Note: The new option to use is `-Dcom.sun.management.jmxremote.rmi.port=XYZ`, see https://tomcat.apache.org/tomcat-9.0-doc/monitoring.html)

Reply:
Done